scripts/build_industry_sector_ratios.py

```diff
@@ -1471,6 +1607,23 @@ def other_industrial_sectors():
 
     params = snakemake.params.industry
 
+    if (
```

Comment:
This should probably throw an error, i prefer if the fractions can not directly be used to fiddle with industry demand